app/plugins/Wallet/js/lifecycle.js

```diff
@@ -46,8 +48,20 @@ function updateAddrTxn(addr) {
 	}, 'update-history');
 }
 
-// Update wallet summary in header
+// Update transaction history and addresses
 addResultListener('update-status', function(wallet) {
+	refreshRate = finalRefreshRate; // slow down after first successful call
+
+	// Show correct lock status.
+	// TODO: If the wallet is encrypted, prompt with a pw.
```

Comment:
What I meant when I wrote the TODO is that when opening the wallet plugin, the user should be prompted to enter their password. Unless it's viable that the plugin is used meaningfully when locked, the user should login on opening, right?